### PR TITLE
remove iat and trust_mark parameters from trust mark status endpoint; #24

### PR DESCRIPTION
**Pull request by zachmann**
To move the discussion on #24 forward I submit a proposal to remove the `iat` and `trust_mark` parameters from the trust mark status endpoint.

### CODE REVIEW

File: openid-federation-1_0.xml [hunk 1]
```diff
@@ -4444,8 +4444,8 @@ Host: openid.sunet.se
 
       <section title="Trust Mark Status" anchor="status_endpoint">
         <t>
-          This enables an Entity to check whether a Trust Mark is still
-          active or not. The query MUST be sent to the Trust Mark issuer.
+          This enables an Entity to check whether a Trust Mark has been issued to
+          an Enity and is still active. The query MUST be sent to the Trust Mark issuer.
```
Comment (peppelinux) [inline openid-federation-1_0.xml:4448]:
```suggestion
          an Enitty and is still active. The query MUST be sent to the Trust Mark issuer.
```

Comment (zachmann) [reply openid-federation-1_0.xml:4448]: Almost correct:
```suggestion
          an Entity and is still active. The query MUST be sent to the Trust Mark issuer.
```

**Comment by selfissued** [comment]
The PR changes the HTTP method from POST to GET, but doesn't update the accompanying example, which current is:
> POST /federation_trust_mark_status_endpoint HTTP/1.1
> Host: op.example.org
> Content-Type: application/x-www-form-urlencoded

> sub=https%3A%2F%2Fopenid.sunet.se%2FRP
> &trust_mark_id=https%3A%2F%2Frefeds.org%2Fsirtfi

Please do so.

File: openid-federation-1_0.xml [hunk 2]
```diff
@@ -4458,7 +4458,7 @@ Host: openid.sunet.se
 
         <section anchor="tm-status-request" title="Trust Mark Status Request">
           <t>
-            The request MUST be an HTTP request using the POST method
+            The request MUST be an HTTP request using the GET method
```
Comment (peppelinux) [inline openid-federation-1_0.xml:4461]: I ask to support both GET and POST to reduce breaking changes in the current implementations.

technically, from an implementation perspective, this is not a big change.

**Review by peppelinux — CHANGES_REQUESTED** [review]

File: openid-federation-1_0.xml (hunk 2 above)
Comment (selfissued) [reply openid-federation-1_0.xml:4461]: @zachmann can you please apply @peppelinux's suggestion about supporting both GET and POST?

Comment (vdzhuvinov) [reply openid-federation-1_0.xml:4461]: If Trust Marks a re no longer being submitted, and we're left only with `sub` and `trust_mark_id`, I don't see reason to support POST in the long term.

**Review by peppelinux — APPROVED** [review]

File: openid-federation-1_0.xml (hunk 2 above)
Comment (peppelinux) [reply openid-federation-1_0.xml:4461]: ok

**Review by vdzhuvinov — APPROVED** [review]

**Review by selfissued — APPROVED** [review]

**Pull request merged**